crates/turborepo-api-client/src/spaces.rs

```diff
@@ -20,14 +20,31 @@ pub struct SpaceClientSummary {
     pub version: String,
 }
 
-#[derive(Default, Debug, Serialize)]
+#[derive(Debug, Serialize)]
+#[serde(rename_all = "camelCase")]
 pub struct SpacesCacheStatus {
-    pub status: String,
-    pub source: Option<String>,
-    pub time_saved: u32,
+    pub status: CacheStatus,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub source: Option<CacheSource>,
+    pub time_saved: u64,
+}
+
+#[derive(Debug, Serialize, Copy, Clone)]
+#[serde(rename_all = "UPPERCASE")]
+pub enum CacheStatus {
```

Comment:
These definitions are copy-pasta'd from `summary/task.rs` just because we don't necessarily need the JSON that gets sent to spaces and the output of dry run to match. If we want to commit to keeping these in lock step, I'm more than happy to do that and have `summary/task.rs` import the definitions from here.